components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -14,6 +14,30 @@
 
 #import <UIKit/UIKit.h>
 
+@class MDCTabBarView;
+
+/**
+ Delegate protocol for MDCTabBarView. Clients may implement this protocol to receive notifications
+ of selection changes by user action in the tab bar.
+ */
+@protocol MDCTabBarViewDelegate <UIScrollViewDelegate>
```

Comment:
Let's put the delegate into its own file. That way clients don't necessarily need to depend on the TabBarView just to be the delegate.

Reply:
Done.